src/components/SkillCreator/SkillViews/ConversationView.js

```diff
@@ -108,16 +101,15 @@ class ConversationView extends Component {
           this.setState({ conversationsData }, () =>
             this.getResponses(++responseNumber),
           );
-        }.bind(this),
-        error: function(err) {
-          console.log(err);
+        })
```

Comment:
@akshatnitd  When this is true `responseNumber + 1 === userInputs.length` then only loaded should be set as true. That is why there are two setState statements. I have put it in an if-else block this will prevent the setState being called twice on the final condition check.
Similar thing in TreeView.js